src/hotspot/share/oops/accessBackend.cpp

```diff
@@ -35,6 +35,11 @@
 namespace AccessInternal {
 // These forward copying calls to Copy without exposing the Copy type in headers unnecessarily
 
+#if defined(TARGET_COMPILER_gcc)
+// Needed to work around bug in gcc causing these symbols to be visible despite -fvisibility=hidden
+#pragma GCC visibility push(hidden)
```

Comment:
does it work for you? because it doesn't hide the arraycopy symbols for me. The explicit visibility(hidden) attribute did work.